### ISSUE TITLE
Schemix CLI

### ISSUE THREAD

**Issue by ridafkih** [ridafkih/schemix#63]
Would it be easier if there was a schemix CLI that was responsible for outputting the `schema.prisma` file?

In this situation, the schemix configuration would be formalized into a file (think `schemix.config.ts` or `schemix.config.js` and would export a default object which would contain a configuration.

After this, you could run a "schemix CLI" that would generate your output file.

```bash
schemix mix
```

Would look in the root directory for a config file, otherwise you would be able to provide it options directly and override options in the config if desired.

```bash
schemix mix --config ./some-other-file.ts --output ./otherly-named-file.prisma
```

**Comment by SwapnilSoni1999** [comment]
would love to take this up 
will try to contribute to this project
Amazing work with Schemix though <3

**Comment by ridafkih** [comment]
Closed in favour of #75, which will track the next major version of schemix which will include these improvements!
